### ISSUE TITLE
Reflection Warnings

### ISSUE THREAD

**Issue by ikitommi** [clojure-goes-fast/clj-async-profiler#7]
`0.2.2` yields the following:

```clj
Reflection warning, clj_async_profiler/server.clj:17:4 - call to method sendResponseHeaders can't be resolved (target class is unknown).
Reflection warning, clj_async_profiler/server.clj:18:10 - reference to field getResponseBody can't be resolved.
Reflection warning, clj_async_profiler/server.clj:19:14 - reference to field getBytes can't be resolved.
Reflection warning, clj_async_profiler/server.clj:19:6 - call to method write can't be resolved (target class is unknown).
Reflection warning, clj_async_profiler/server.clj:20:6 - reference to field close can't be resolved.
Reflection warning, clj_async_profiler/server.clj:33:29 - call to method lastIndexOf can't be resolved (target class is unknown).
Reflection warning, clj_async_profiler/server.clj:33:3 - call to method substring can't be resolved (target class is unknown).
Reflection warning, clj_async_profiler/server.clj:36:28 - reference to field getName can't be resolved.
Reflection warning, clj_async_profiler/server.clj:37:27 - reference to field getRequestMethod can't be resolved.
Reflection warning, clj_async_profiler/server.clj:38:16 - reference to field length can't be resolved.
Reflection warning, clj_async_profiler/server.clj:39:11 - reference to field getResponseHeaders can't be resolved.
Reflection warning, clj_async_profiler/server.clj:40:7 - call to method add can't be resolved (target class is unknown).
Reflection warning, clj_async_profiler/server.clj:41:7 - call to method add can't be resolved (target class is unknown).
Reflection warning, clj_async_profiler/server.clj:43:5 - call to method sendResponseHeaders can't be resolved (target class is unknown).
Reflection warning, clj_async_profiler/server.clj:47:40 - reference to field getResponseBody can't be resolved.
Reflection warning, clj_async_profiler/server.clj:56:60 - reference to field getRequestURI can't be resolved.
Reflection warning, clj_async_profiler/server.clj:60:21 - reference to field getResponseHeaders can't be resolved.
Reflection warning, clj_async_profiler/server.clj:60:15 - call to method add can't be resolved (target class is unknown).
Reflection warning, clj_async_profiler/server.clj:74:25 - call to method stop on java.lang.Object can't be resolved (no such method).
Reflection warning, clj_async_profiler/core.clj:77:8 - call to method addURL on java.lang.Object can't be resolved (no such method).
Reflection warning, clj_async_profiler/core.clj:162:16 - call to method getDeclaredMethod on java.lang.Object can't be resolved (no such method).
Reflection warning, clj_async_profiler/core.clj:163:5 - call to method invoke can't be resolved (target class is unknown).
Reflection warning, clj_async_profiler/core.clj:183:10 - call to method loadAgentPath can't be resolved (target class is unknown).
Reflection warning, clj_async_profiler/core.clj:258:22 - call to method contains on java.lang.Object can't be resolved (no such method).
```

**Comment by alexander-yakushev** [comment]
Thanks! I fixed all but one in `0.2.3-SNAPSHOT`. The one remaining is a place where I don't know the exact classloader class, so, unfortunately, it would stay.

**Comment by ikitommi** [comment]
thanks for the quick reso.
